build.gradle

```diff
@@ -148,33 +149,60 @@ tasks.register('checkAkitInstall', JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 }
 
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-    from new File(rootProject.rootDir, '.scripts/post-checkout')
-    into { new File(rootProject.rootDir, '.git/hooks') }
-    filePermissions {
-        unix(0775)
-    }
-    doLast {
-        println("---Added Post Checkout Git Hook Script---")
+def ensureSymlink(link, target) {
+    if (!Files.isSymbolicLink(link)) {
+        if (Files.exists(link)) {
+            Files.delete(link)
+        }
+
+        Files.createSymbolicLink(link, target)
+
+        def relative = { f -> rootDir.toPath().relativize(f) }
```

Comment:
maybe move it to outside function, "getPathFromRootDir(link)"

Reply:
I personally disagree, but thinking about it; what do you think of just foregoing any printing there at all? Most tasks aren't so verbose

Reply:
what do you mean? 
Also about the outside function, when its outside it can be used more easily in other tasks and more readable and informative

Reply:
I mean what if we don't print the whole 'Installed symbolic link ...'; it's not that necessary